.github/workflows/main.yml

```diff
@@ -0,0 +1,42 @@
+name: Linting
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
Hey, @dhruvrajsinh-7. Our master branch is named "master" not "main" so make the changes as per it.